src/viur/core/modules/user.py

```diff
@@ -221,8 +221,15 @@ def __init__(self, moduleName, modulePath, userModule):
         super().__init__(moduleName, modulePath)
         self._user_module = userModule
 
+    def can_handle(self, user: db.Entity) -> bool:
+        return True
 
-class UserPassword(UserAuthentication):
+
+class UserPrimaryAuthentication(UserAuthentication, abc.ABC):
+    """Abstract class for all primary authentication methods."""
```

Comment:
Why does this class exist without a member or a method ?
```suggestion
class UserPrimaryAuthentication(UserAuthentication, abc.ABC):
    """Abstract class for all primary authentication methods."""
    registrationEnabled = False
```
We can add for example `registrationEnabled = False` or `def getAuthMethodName`

Reply:
> Why does this class exist without a member or a method ?
> We can add for example `registrationEnabled = False` or `def getAuthMethodName`

Yes, please! Make a proposal or edit the PR. I've added the class only to clearly identify primary authentication providers.

Reply:
ok.